website/content/en/highlights/2022-02-08-0-20-0-upgrade-guide.md

```diff
@@ -38,3 +39,25 @@ it.  For sets, however, we would simply clear their values and thus report a set
 behavior was not consistent with how we expired other metric types, and additionally, could still
 lead to a growth of unique series in a scrape, over time, as we would clear the set but not stop
 reporting it.
+
+#### New metrics behavior for `route` transform {#metrics-route-transform}
+
+Previously, a `route` transform was internally expanded into a separate
+transform for each defined output. Each of these expanded transforms then
+emitted its own metrics. Now, with improved internal support for components with
+multiple outputs, this expansion mechanism has been removed.
+
+A single `route` transform will now emit one set of metrics. For the following
+metrics, specific output information will be recorded as a metric tag `output`.
+
+- `component_sent_events_total`
+- `component_sent_event_bytes_total`
+- `events_discarded_total`
+
+For example, for a `route` transform `foo` with an output `first` configured,
+the `events_discarded_total` metric will now look like the following:
```

Comment:
```suggestion
Previously, a `route` transform was internally expanded into a separate
transform for each defined route. Each of these expanded transforms then
emitted its own metrics. Now, with improved internal support for components with
multiple outputs, this expansion mechanism has been removed.

A single `route` transform will now emit one set of metrics. For the following
metrics, specific route information will be recorded as a metric tag `output`.

- `component_sent_events_total`
- `component_sent_event_bytes_total`
- `events_discarded_total`

For example, for a `route` transform `foo` with a route `first` configured,
the `events_discarded_total` metric will now look like the following:
```

I can see users being somewhat confused about the mixing of route and output terminology for the `route` transform. We could update the `route` transform docs to make it more clear that routes now map to outputs.